flash_attn/cute/flash_bwd_sm100.py

```diff
@@ -900,12 +921,10 @@ def kernel(
         else:
             sdV = storage.sdO.get_tensor(sdKV_layout, dtype=self.dv_dtype)
             sdK = storage.sQ.get_tensor(sdKV_layout, dtype=self.dk_dtype)
-        assert cute.size_in_bytes(self.do_dtype, sdO_layout) >= cute.size_in_bytes(
```

Comment:
keep the asserts, testing against buffer size?

Reply:
just moved them closer to allocation